README.md

```diff
@@ -8,7 +8,41 @@ A library to **sub**mit e**xt**rinsics to a [substrate](https://github.com/parit
 
 ## Usage
 
-See [examples](./examples).
+### Downloading metadata from a Substrate node
+
+Use the `subxt-cli` CLI tool to download the metadata for your target runtime from a Substrate node.
+
+```bash
+cargo run -p subxt-cli -- metadata -f bytes > metadata.scale
```

Comment:
Sure would be good to have a release on crates so we could write `cargo install subxt-cli` here. :/

Reply:
Absolutely!

Reply:
I suppose there's nothing stopping us releasing a `subxt-cli` `0.1`

Reply:
https://github.com/paritytech/subxt/pull/320